gui/src/components/mainInput/ActiveFileIndicator.tsx

```diff
@@ -5,7 +5,11 @@ import { useCallback, useContext, useEffect } from "react";
 import { X } from "lucide-react";
 import { setActiveFilePath } from "@/redux/slices/uiStateSlice";
 
-export default function ActiveFileIndicator() {
+interface ActiveFileIndicatorProps {
+  isFirstInSession?: boolean;
```

Comment:
I think instead of using prop. We can just check the history within active file indicator. See this PR: https://github.com/trypear/pearai-submodule/pull/181/files#diff-ea630c971a646645666575d7bbfe531ea6c00a5cb7108f3dc66ded51fccfd963R20

This PR I made above i will not be merging. Was playing around with some stuff but not good. However I did do the history stuff in there to determine first chat input, which may be worth using instead of prop drilling. Also we should grey out the thing, with unchangeable text if conversation is prompted.